### DISCUSSION TITLE
How to hide table reference hover preview?

### DISCUSSION THREAD

**Discussion by aborruso** [quarto-dev/quarto-cli#7705] [Q&A] [answered]
### Description

Hi,
this information is probably written [here](https://quarto.org/docs/reference/formats/html.html#tables), but I don't see it.

How to hide table reference hover preview?

![image](https://github.com/quarto-dev/quarto-cli/assets/30607/298e0efe-f2e2-46d0-a6e0-1d117ad09c2d)

Thank you

**Comment by mcanouil** [comment]
Did you look for "hover" in the documentation? The answer is the second item.

**Reply by aborruso** [reply]
> Did you look for "hover" in the documentation? The answer is the second item.

Please don't hate me. I looked here, on the reference page for the HTML, and I have:

- `citations-hover`
- `footnotes-hover`

I do not see something like `reference-hover`. Could you write me what's the right page I must see?

Excuse me and thank you

**Reply by mcanouil** [reply, marked as answer]
Hum, I thought `citations-hover` also controlled references, but apparently not, meaning there is currently no easy way to remove the hover. 

I opened an enhancement issue to track this:
- #7707

**Reply by aborruso** [reply]
> Hum, I thought citations-hover also controlled references, but apparently not, meaning there is currently no easy way to remove the hover.

I'm glad I hadn't written nonsense this time. It seems important to me to have this option, because in the case of my example it's just redundant to have the preview.

**Reply by mcanouil** [reply]
Note that you would not be able to control on reference level, so disabling it, would disable it for every reference to float objects.

**Reply by mcanouil** [reply]
FYI:
- https://github.com/quarto-dev/quarto-cli/issues/7707#issuecomment-1828658684

**Reply by aborruso** [reply]
Thank you very much @mcanouil
